openpathsampling/experimental/simstore/test_sql_backend.py

```diff
@@ -1,6 +1,6 @@
 from .sql_backend import *
 import pytest
-
+sqla = pytest.importorskip("sqlalchemy")
```

Comment:
```suggestion
```

(or `import sqlalchemy as sql`)

1. `importorskip` shouldn't be necessary. If running SimStore tests, SQLAlchemy is a hard requirement. Ignoring these tests (unless you're running from the `experimental` directory) is handled by our `pytest.ini`.
2. You already have the import (as `sql`) from `from .sql_backend import *`. Actually, it looks like the more common short name for SQLAlchemy is `sa`, but lets stay consistent within the project.

(Some stats on "as" names [here](https://import-as.github.io/import/sqlalchemy.html), though I have no idea where the numbers come from. It seems to cover all short names with at most four characters. (No `paths`, but it does have [`toys`](https://import-as.github.io/import/openpathsampling.engines.toy.html) for the toy engine!)